default-fields/0000-default-fields.md

```diff
@@ -28,9 +233,29 @@ Why should we *not* do this?
 # Alternatives
 [alternatives]: #alternatives
 
-What other designs have been considered? What is the impact of not doing this?
+Just stick with builders and constructor functions.
 
 # Unresolved questions
 [unresolved]: #unresolved-questions
 
-What parts of the design are still TBD?
+## `const`
+
+Is `const` the right feature to base off?
+It means field values are just as constrained as `const` is now,
+but also prevents struct definitions from becomes bloated by complex logic that
+maybe belongs in a builder instead.
```

Comment:
I would move this to alternatives (and phrased a bit differently). Also consider runtime overhead/expectations of struct literal vs function call